chrome/content/zotero/bindings/itembox.xml

```diff
@@ -2476,6 +2490,8 @@
 							return !hideTransforms;">
 						<menuitem label="&zotero.item.creatorTransform.nameSwap;"
 							oncommand="document.getBindingParent(this).swapNames();"/>
+						<menuitem label="&zotero.item.textTransform;"
+							oncommand="document.getBindingParent(this).textTransform(document.popupNode, 'title')"/>
```

Comment:
Any suggestions how to name it?
 * "Title Case" (which we already have...)
 * "Fix Cases"
 * "Fix Caps"
 * ...